features/apimgt/org.wso2.carbon.apimgt.publisher.feature/pom.xml

```diff
@@ -205,22 +205,60 @@
                 <version>1.5.0</version>
                 <executions>
                     <execution>
-                        <id>npm install (initialize)</id>
+                        <id>npm install(mvn)</id>
                         <goals>
                             <goal>exec</goal>
                         </goals>
                         <phase>initialize</phase>
                         <configuration>
-                            <environmentVariables>
-                                <PUPPETEER_SKIP_CHROMIUM_DOWNLOAD>
-                                    true
-                                </PUPPETEER_SKIP_CHROMIUM_DOWNLOAD>
-                            </environmentVariables>
-                            <workingDirectory>src/main/resources/publisher</workingDirectory>
+                            <workingDirectory>../</workingDirectory>
+                            <executable>${npm.executable}</executable>
+                            <arguments>
+                                <argument>ci</argument>
+                            </arguments>
+                        </configuration>
+                    </execution>
+                    <execution>
+                       <id>lerna clean</id>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                        <phase>initialize</phase>
+                        <configuration>
+                            <workingDirectory>../</workingDirectory>
```

Comment:
Why we change the working directory one level up ? isn't `package.json` file at the same level where this `pom.xml` is ?